app/uk/gov/hmrc/tariffclassificationfrontend/controllers/CaseController.scala

```diff
@@ -190,7 +195,7 @@ class CaseController @Inject()(verify: RequestActions,
   private def showActivity(c: Case, f: Form[ActivityFormData])
                           (implicit request: AuthenticatedRequest[AnyContent]): Future[HtmlFormat.Appendable] = {
     for {
-      events <- eventsService.getEvents(c.reference, NoPagination())
+      events <- eventsService.getFilteredEvents(c.reference, NoPagination(),Some(EventType.values.diff(Set(EventType.SAMPLE_STATUS_CHANGE))))
```

Comment:
I would prefer to define the positive one which means define all the ones you want. But quite clever any way ;-)

Reply:
I did it like that to start with and maybe it is clearer actually, but I thought this might be better as firstly it makes the line really long if all are listed, and secondly I thought for future safety, if new types are added they wouldn't automatically be shown on activity feed unless they were added here, whereas this way keeps it futureproof as it's always 'everything except sample status'

Reply:
I guess we will have the problem either way. In this case if we add something we don't want in this specific list we will add it for anyway. I prefer positive code but I agree to keep this one..